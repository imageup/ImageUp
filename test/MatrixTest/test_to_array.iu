function main() -> int
{
	matrix : arr(10, 1);
	int : i;
	for(i = 0; i < RowLen(arr); i=i+1)
	{
		arr[i][0] = i;
	}
	for(i = 0; i < RowLen(arr); i=i+1)
	{
		printf(arr[i][0]);
	}
	return 0;
}