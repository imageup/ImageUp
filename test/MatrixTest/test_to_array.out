llc -relocation-model=pic test.ll > test.s ;gcc -o test.exe test.s lib.c `pkg-config --cflags --libs opencv`;./test.exe
Makefile:14: recipe for target 'testing' failed
